atm.js

```diff
@@ -1,21 +1,87 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+$(document).ready(function(){
```

Comment:
this is not wrong, but `document.ready` is unecessary here because the script tag is just before the closing body tag. The DOM will have loaded by the time your script is called.